samples/js/browser/avatar/js/basic.js

```diff
@@ -68,6 +68,18 @@ function setupWebRTC(iceServerUrl, iceServerUsername, iceServerCredential) {
         }
     }
 
+    // Listen to data channel, to get the event from the server
+    peerConnection.addEventListener("datachannel", event => {
+        const dataChannel = event.channel
+        console.log("connecting to data channel" + dataChannel.label)
+        dataChannel.onmessage = e => {
+            console.log("[" + (new Date()).toISOString() + "] Event received: " + e.data)
```

Comment:
Can you update "Event received" to "WebRTC event received"?
Because we also have other events from non-WebRTC channel.